### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by NamrataRaikwar2002**
-video listing page

Screenshot from 2022-04-11 14-47-55

### CODE REVIEW

File: src/Component/Drawer/Drawer.jsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import './Drawer.css'
+export const Drawer = () => {
+  return (
+    <>
```
Comment (gadgetfather) [inline src/Component/Drawer/Drawer.jsx:4]: You can remove React-fragment if you already have one parent element

**Review by gadgetfather — APPROVED** [review]
- Nice UI 
- Suggestion: You can reduce the box shadow for the navbar
Great work !

File: src/Component/Drawer/Drawer.jsx (hunk 1 above)
Comment (NamrataRaikwar2002) [reply src/Component/Drawer/Drawer.jsx:4]: i will do the changes

**Review by mohammed-sas — APPROVED** [review]
The video listing screens looks fine

**Pull request merged**